static/tabs/setup.html

```diff
@@ -15,16 +15,15 @@
                     id="config-upload-form"
                 >
                     <div class="card">
-                        <div class="card-header fs-4">
-                            Upload AnkerMake Login File
-                        </div>
+                        <div class="card-header fs-4">Upload AnkerMake Login File</div>
 
                         <div class="card-body input-group">
                             <input
                                 class="form-control"
                                 type="file"
                                 id="loginFile"
                                 name="login_file"
+                                accept=".json"
```

Comment:
When testing, I often keep a `login.bak` or similar file on hand. I don't see any reason why we should artificially restrict what the file can be called - we should just send it, and let the backend tell us if it works or not :)

Reply:
✅